platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -273,6 +273,10 @@ NS_ASSUME_NONNULL_BEGIN
  Touch frameworks. On the other hand, static annotation images use less memory
  and draw more quickly than annotation views.
  
+ The user location annotation view can also be customized via this method. When
+ `annotation` is `MGLUserLocation`, return an instance of a custom
+ `MGLUserLocationAnnotationView` subclass.
```

Comment:
Sounds good. To be pedantic:

s/is MGLUserLocation/is an instance of MGLUserLocation (or equal to the map view's userLocation property)/